bindings/python/src/pipeline/CommonBindings.cpp

```
@@ -1,7 +1,11 @@
 #include "CommonBindings.hpp"
 
+#include <pybind11/detail/common.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)